devdoc/constbuffer_array_requirements.md

```diff
@@ -98,6 +98,34 @@ Note: `constbuffer_array_create_from_buffer_index_and_count` does not increment
 
 **SRS_CONSTBUFFER_ARRAY_42_016: [** If any error occurs then `constbuffer_array_create_from_buffer_index_and_count` shall fail and return `NULL`. **]**
 
+### constbuffer_array_create_from_start_and_end
+
+```c
+MOCKABLE_FUNCTION(, CONSTBUFFER_ARRAY_HANDLE, constbuffer_array_create_from_start_and_end, CONSTBUFFER_ARRAY_HANDLE, original, uint32_t, start_buffer_index, uint32_t, start_buffer_offset, uint32_t, start_buffer_size, uint32_t, end_buffer_index, uint32_t, end_buffer_offset, uint32_t, end_buffer_size);
```

Comment:
>end_buffer_index [](http://example.com/codeflow?start=227&length=16)

I think we should have a buffer count like we do in `constbuffer_array_create_from_buffer_index_and_count` instead of the end index. Just for consistency.

Reply:
done